### ISSUE TITLE
Notification Preferences feature does not work

### ISSUE THREAD

**Issue by kapsali29** [QualiChain/fe#444]
@mmilaprat the notification preference frontend feature has some issues. More specifically when someone tries to create a new notification preference something weird happens with the location definition... Could you please take a look?

**Comment by mmilaprat** [comment]
can you give me more details? with this information I cannot reproduce it

**Comment by kapsali29** [comment]
Yes sure... It is not clear that the end user needs to use the location drop down menu to select a location. So if someone tries to write a location without selecting one from the dropdown list the Notification preference feature is not properly used. 

![1](https://user-images.githubusercontent.com/29201775/125626792-07da2063-e78b-4aa1-a4a4-e00813bb6d7d.PNG)

![2](https://user-images.githubusercontent.com/29201775/125626882-a10d68bc-6d66-4ed6-94be-b70c58f7395b.PNG)

**Comment by mmilaprat** [comment]
you need to press the enter or the comma key to "accept" your manual imput, then the location will appear over a "grey" box, if it is not over the "grey box" it is not valid.

the plus button is a wizard to find some cities, but as we cannot find all the cities with the wizard, the field is open to a manual imput.

**Comment by mmilaprat** [comment]
Can you check if the latest update has solve the reported issue with the location field?
